CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Added
```

Comment:
Clickhouse backend support should be in this section. I think we've missed a rebase failure :sweat_smile: